### PR TITLE
Fix spelling errors

### PR DESCRIPTION
**Pull request by dforsi**
This PR applies automatic edits done with the following command:
codespell --summary --ignore-words-list mot,ned,parm,parms,upto --skip cmake --write-changes

Except the changes in component_metadata/actuators.schema.json all edits are done in texts means for humans, such as descriptions and comments.

### CODE REVIEW

File: message_definitions/v1.0/common.xml [hunk 1]
```diff
@@ -5225,7 +5225,7 @@
       <field type="uint8_t" name="mission_type" enum="MAV_MISSION_TYPE">Mission type.</field>
     </message>
     <message id="48" name="SET_GPS_GLOBAL_ORIGIN">
-      <description>Sets the GPS co-ordinates of the vehicle local origin (0,0,0) position. Vehicle should emit GPS_GLOBAL_ORIGIN irrespective of whether the origin is changed. This enables transform between the local coordinate frame and the global (GPS) coordinate frame, which may be necessary when (for example) indoor and outdoor settings are connected and the MAV should move from in- to outdoor.</description>
+      <description>Sets the GPS coordinates of the vehicle local origin (0,0,0) position. Vehicle should emit GPS_GLOBAL_ORIGIN irrespective of whether the origin is changed. This enables transform between the local coordinate frame and the global (GPS) coordinate frame, which may be necessary when (for example) indoor and outdoor settings are connected and the MAV should move from in- to outdoor.</description>
```
Comment (hamishwillee) [inline message_definitions/v1.0/common.xml:5228]: This is questionable, but neither is wrong.

File: component_metadata/actuators.schema.json [hunk 2]
```diff
@@ -17,17 +17,17 @@
             "properties": {
                 "min": {
                     "type": "number",
-                    "minium": -1,
+                    "minimum": -1,
```
Comment (hamishwillee) [inline component_metadata/actuators.schema.json:20]: @bkueng I want to accept this PR. Will these fixes to the schema cause you any problems?

**Comment by hamishwillee** [comment]
Thanks @dforsi - looks great. Just checking on the schema changes and then should be able to merge.

**Review by hamishwillee — APPROVED** [review]

File: component_metadata/actuators.schema.json (hunk 2 above)
Comment (bkueng) [reply component_metadata/actuators.schema.json:20]: This looks good. I don't think it will.

**Comment by hamishwillee** [comment]
Merged! Thanks for the fix @dforsi

**Pull request merged**